application/routes/audio.py

```
@@ -35,22 +35,27 @@
 from .. import api
 from ..codes import ConnectionType
 from ..features.Audio import AUDIO_PORT
-
+import logging
+logger = logging.getLogger(__name__)
 
 @api.route('/audio', methods=['POST'])
 def start_audio():
```

Ruff

Missing return type annotation for public function `start_audio`

(ANN201)